src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/utils/Environment.scala

```diff
@@ -44,6 +45,8 @@ val baseDomainOrg="manage-org.perftest.platform.hmcts.net"
   val maxThinkTimePROB = 10//310
   val constantthinkTime = 5
   val HttpProtocol = http
+  val docId1 = "7228f9f2-c8c8-48e1-aa34-817e767a5c13"
```

Comment:
I'm still unsure about these, if you're uploading a document during the creation process then the doc Id will change no?

Reply:
Made it dynamic but I think there's a bug

Reply:
Ticket needs raising then

Reply:
Will do

Reply:
Fixed